index.js

```diff
@@ -3,5 +3,15 @@ const banks = require('./banks');
 module.exports = {
   getBanks: function() {
     return banks;
+  },
+  getBank: code => {
+  	let OneBank = null
+  	let allBanks = [...banks]
```

Comment:
There's no point spreading this into another variable, you're never going to mutate it so it's fine to use the `banks` array as it is.